Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -198,6 +198,9 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 
 		protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
+			if (_disposed)
```

Comment:
Ive seen this happening on image button label and frame (all fastrenderers). I would try creating a repo on the first prerelase containing fastrenderers (it had zero? dispose checks on any of the fast renderers). And it was crashing a lot because of accessing disposed objects